python-lib/dku_io_utils/recipe_config_loading.py

```diff
@@ -271,3 +274,29 @@ def reorder_column_list(column_list_to_reorder, reference_column_list):
         if column_name in column_list_to_reorder:
             reordered_list.append(column_name)
     return reordered_list
+
+
+def parse_gpu_devices(gpu_devices):
+    """Check the custom python MULTISELECT for GPU devices
+
+    Args:
+        gpu_devices (list): List of GPU number or ["no_gpu_available"]
+
+    Raises:
+        PluginParamValidationError:
+            If more than 1 GPU are selected (for now we support only one GPU)
+            If selected GPU is not an integer
+
+    Returns:
+        List of a single GPU (we may support multiple later) or None
+    """
+    if len(gpu_devices) == 0:  # nothing selected
+        return None
+    elif len(gpu_devices) > 1:
+        raise PluginParamValidationError("Only 1 GPU device can be selected")
+    elif gpu_devices[0] == "no_gpu_available":
+        return None
```

Comment:
```suggestion
        raise PluginParamValidationError("No GPU available, please check your CUDA installation or deactivate the 'Use GPU' parameter"
```

Reply:
alright will make an error, i wasn't sure if we wanted to fail in this case